tools/gn

```diff
@@ -139,6 +139,10 @@ def to_gn_args(args):
     elif args.target_os == 'fuchsia':
         gn_args['target_os'] = 'fuchsia'
         gn_args['flutter_enable_legacy_fuchsia_embedder'] = args.fuchsia_legacy
+    elif args.target_os == 'winuwp':
+        gn_args['target_os'] = 'winuwp'
+        # Enabling angle_is_winuwp is dependent on an in-flight patch to https://chromium.googlesource.com/angle/angle to update UWP support
```

Comment:
Can you link to the PR?

Reply:
I haven't yet opened the PR so maybe we should revise the comment

Reply:
I'm just going to remove the comment; this seems like something we should probably set in a buildroot .gni file based on target_os. We'll see whether there's a good hook point for that once the ANGLE change is in.

Reply:
SG